### PR TITLE
Operation Center start

### PR DESCRIPTION
**Pull request by kevin-christian**

### CODE REVIEW

File: OperationCenter/OperationCenter.h [hunk 1]
```diff
@@ -10,6 +10,8 @@
 #include <sstream>
 #include "Commons.h"
 #include "DataProcessor.h"
+#include "Parser.h"
+#include "DataStorage.h"
```
Comment (YangXiaozhou) [inline OperationCenter/OperationCenter.h:14]: why do you need to include DataStorage in O/C?

File: DataProcessor/DataProcessor.cpp [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+string DataProcessor::showWelcomeMessage(){
+	string welcomeMessage;
+	welcomeMessage = "Welcome to BlinkList";
+	return welcomeMessage;
+}
+
+string DataProcessor::addTask(Data task){
+	dataStorage.addTaskList(task); //dataStorage is a DataStorage object initialized in O/C
```
Comment (YangXiaozhou) [inline DataProcessor/DataProcessor.cpp]: Is there such an API in Angie's DataStorage header file?

File: DataProcessor/DataProcessor.cpp [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+		out << "'" << taskUniqueNumber << " " << taskDescription << "' is deleted successfully" << endl;
+	} else {
+		out << "'" << taskUniqueNumber << "' does not exist" << endl; 
+	}
+	string deleteMessage;
+	deleteMessage = out.str();
+	return deleteMessage;
+}
+
+string DataProcessor::display(Time startTime, Time endTime){
```
Comment (YangXiaozhou) [inline DataProcessor/DataProcessor.cpp]: Time object is now update to either TimeMacro or TimeMicro object.

**Comment by YangXiaozhou** [comment]
@ChunhanYu @angelitasasmita review the pull request please.

File: DataProcessor/DataProcessor.cpp (hunk 2 above)
Comment (kevin-christian) [inline DataProcessor/DataProcessor.cpp]: i dont know, when i made it she has not made her DataStorage API, so i just wrote it first

**Comment by YangXiaozhou** [comment]
yeah, seems okay to merge.

**Pull request merged**